internal/ir/xds.go

```diff
@@ -1640,7 +1640,9 @@ type DestinationSetting struct {
 	// Lower priority endpoints will be used only if higher priority levels are unavailable.
 	Priority *uint32 `json:"priority,omitempty"`
 	// Protocol associated with this destination/port.
-	Protocol  AppProtocol            `json:"protocol,omitempty" yaml:"protocol,omitempty"`
+	Protocol AppProtocol `json:"protocol,omitempty" yaml:"protocol,omitempty"`
+	// Hostname refers to the endpoint's hostname
```

Comment:
apologies for just noticing this
the `Hostname` is per `Endpoint` in the API, so it should be under `Endpoint` where it was before

Reply:
Okay, back to the original version.